wikidev/wikidev/views.py

```diff
@@ -103,33 +123,24 @@ def get(self, request):
             article_id, article_details = [*article_details.items()][0]
             title = article_details['title']
 
-            body = [
-                onem.menus.MenuItem(
-                    article_details.get('extract', '').split('==')[0].strip(),
-                    is_option=False
-                )
-            ]
+            body = MenuItem(type=MenuItemType.content,
```

Comment:
Not a functional problem, but `body` should be a list as it was before. Not a problem tho.

Reply:
I am casting it to a list below, on line 131:
`menu = Menu(body=[body], header='random',footer='Reply MENU')`